pkgs/servers/mastodon/source.nix

```diff
@@ -1,12 +1,13 @@
 # This file was generated by pkgs.mastodon.updateScript.
-{ fetchFromGitHub, applyPatches }: let
+{ fetchFromGitHub, applyPatches }:
+applyPatches {
+  version = "4.1.9";
   src = fetchFromGitHub {
     owner = "mastodon";
     repo = "mastodon";
     rev = "v4.1.9";
```

Comment:
Since the version ends up here, do you want to either `rec` or `let` it and use `rev = "v${version}"` here as well?

Reply:
Done in 300a6fda7724be412f55c286cd209bb6b0301ac3.